src/api/status/public/index.html

```diff
@@ -36,7 +36,7 @@
     <link href="https://fonts.googleapis.com/icon?family=Material+Icons+Round" rel="stylesheet" />
     <!-- CSS Files -->
     <link id="pagestyle" href="assets/css/material-dashboard.css?v=3.0.0" rel="stylesheet" />
-    <link rel="stylesheet" type="text/css" href="./css/site.css" />
+    <link rel="stylesheet" href="css/site.css" />
```

Comment:
Why remove the `type`?

Reply:
Oh I just want to make it similar with the local style sheet below  :)